Assets/Puzzle3.cs

```diff
@@ -0,0 +1,175 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
+using System.Linq;
+using UnityEngine.Rendering;
+
+public class Puzzle3 : MonoBehaviour
+{
+    [SerializeField]
+    public Button[] oliverButtons;
+    [SerializeField]
+    public Button[] wendyButtons;
+    [SerializeField]
+    public GameObject BasementDoor;
+
+    public string correctOliver = "OLIVER";
+    public string correctWendy = "WENDY";
+
+    private Button selectedButton = null;
+
+    private void Start()
+    {
+        Debug.Log("Oliver Buttons Count: " + oliverButtons.Length);
+        Debug.Log("Wendy Buttons Count: " + wendyButtons.Length);
```

Comment:
need to clean up all these debug logs in these files so it doesn't spam the logs and important ones get missed.